### PR TITLE
Updated go version to 1.19 in make file of jx-admin

### PR DESCRIPTION
**Pull request by PratikforCoding**
Signed-off-by: PratikforCoding <kotalpratik@gmail.com>

### CODE REVIEW

**Comment by PratikforCoding** [comment]
related to jenkins-x/jx#8433

File: Makefile [hunk 1]
```diff
@@ -13,7 +13,7 @@ ORG := jenkins-x-plugins
 ORG_REPO := $(ORG)/$(NAME)
 RELEASE_ORG_REPO := $(ORG_REPO)
 ROOT_PACKAGE := github.com/$(ORG_REPO)
-GO_VERSION := 1.17.9
+GO_VERSION := 1.19.3
```
Comment (ankitm123) [inline Makefile:16]: Change to: https://github.com/jenkins-x/jx/blob/7987112f6727f6072aaac2d94eea787f77a99efb/Makefile#L18

Comment (PratikforCoding) [reply Makefile]: changed it @ankitm123

**Comment by ankitm123** [comment]
Also need to change this part: https://github.com/jenkins-x-plugins/jx-admin/blob/9ac6b5394b8ce7535a2c08185b4a1747d22e3eb8/.github/workflows/jenkins-x-release.yaml#L46-L53 
to https://github.com/jenkins-x/jx/blob/0caa3ac8af5d889987c810efb27b38e5b8e72512/.github/workflows/jenkins-x-release.yaml#L39-L47

Also this: https://github.com/jenkins-x-plugins/jx-admin/blob/main/.github/workflows/jenkins-x/upload-binaries.sh needs to be changed to be like this: https://github.com/jenkins-x/jx/blob/main/.github/workflows/jenkins-x/upload-binaries.sh (notice that syft is already installed in the custom image that is being used)

**Comment by PratikforCoding** [comment]
@ankitm123 done those things... thanks a lot for helping!!

**Pull request merged**